CONTRIBUTING.md

```diff
@@ -205,36 +208,37 @@ To make a release as a core contributor, you will need:
 
-- increment the version number of all packages, without pushing to origin.  This will also create a release tag.
+- increment the version number of all packages, and create a local commit, without pushing to origin.  This will also create a release tag.
   - `pnpm lerna version --no-commit-hooks --no-push 7.0.0`
 
-- stage any files changed by the version command.
-  - `git add packages/*/package.json`
-  - `git add lerna.json`
-`
-
-- create new commit on the release branch
-  - `git commit -m "Release v7.0.0"`
-
 - Push the release branch and the release tag.
   - `git push origin mf/release-7.0.0 --follow-tags`
-  - The tag will trigger the Github [release](https://github.com/Turfjs/turf/blob/master/.github/workflows/release.yml) action which you can view the status of at - https://github.com/Turfjs/turf/actions.  If successful, a new [version](https://www.npmjs.com/package/@turf/turf?activeTab=versions) of all turf packages will have been published on NPM.
-
-- If the release action was successful, now create a Pull Request for the release and then merge it.
-  - A link to create the PR should be in the output of the push command
-  - Use the version number for the PR title (e.g. v7.0.0).
-  - If a link isn't provided, just create a PR that merges your release branch to master.  Here is an example - https://github.com/Turfjs/turf/pull/2615.
-  - Don't forget to merge your release PR!
+  - Pushing the tag will trigger the Github [release](https://github.com/Turfjs/turf/blob/master/.github/workflows/release.yml) action which you can view the status of at - https://github.com/Turfjs/turf/actions.  If successful, a new [version](https://www.npmjs.com/package/@turf/turf?activeTab=versions) of all turf packages will have been published on NPM.
+
+- If the release action was not successful
+  - commit any changes to master to fix it.
+    - Make a [prerelease](#prerelease) if helpful to make sure the release action is successful.
+  - Then reset by deleting your tag and branch locally and remotely.
+    - `git push --delete origin v7.1.0`
+    - `git tag --delete v7.1.0`
+    - `git push -d origin mf/release-7.0.0`
+    - `git branch -d mf/release-7.0.0`
+  - Now redo the steps above starting with creating "A clean local copy of the Turf Github repository"
+
+- If the release action was successful, now create a Pull Request for the release to update master.
+  - You may be given a link in the output of the branch/tag push command to create the PR.
+  - If you don't get this message, just go to https://github.com/Turfjs/turf/pulls and you should be prompted at the top to create a PR for this new branch you just pushed.
+  - If that prompt doesn't appear, then just create a new pull request from the PR page and make sure the title is the version number e.g. `v7.0.0` and that it is merging your release branch -> to master.
+  - Here is an example PR - https://github.com/Turfjs/turf/pull/2615.
+  - Get approval for the release PR and merge it!
```

Comment:
```suggestion
  - If the publish action was successful, get approval for the release PR and merge it!
```

and then maybe delete the next line entirely?

Reply:
yep